### ISSUE TITLE
Prepare training data set

### ISSUE THREAD

**Issue by chameerawijebandara** [chameerawijebandara/Satellite-Image-Analyzer-#2]
Crete training data set with sufficient size (positive and negative)

**Comment by HBMDDS** [comment]
I got a working code to train a NN. Just need to adjust some values and give training samples

**Comment by chameerawijebandara** [comment]
Great

**Comment by chameerawijebandara** [comment]
Could you please  upload sample trang data files here

**Comment by HBMDDS** [comment]
Finished the code to crop samples of trees from images. We can click on a point on the image and small box (8x8) will be drawn around that point. Then if we want we can save that 8x8 image by right clicking.

**Comment by chameerawijebandara** [comment]
Now we can build a suffisent dataset and start training the model  :)

**Comment by HBMDDS** [comment]
Yes I think and I will upload codes (for crop sample image) later..

**Comment by chameerawijebandara** [comment]
Any update machan,
need to see some sample images
